packages/react-components/src/components/VideoTile.tsx

```diff
@@ -156,32 +156,34 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           </Stack>
         )}
 
-        <Stack
-          horizontal
-          className={mergeStyles(
-            isVideoRendered ? videoHint : disabledVideoHint,
-            // when video is being rendered, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
-          )}
-        >
-          <Stack>
-            {displayName && (
-              <Text
-                className={mergeStyles(displayNameStyle, {
-                  color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
-                })}
-              >
-                {displayName}
-              </Text>
+        {(displayName || showMuteIndicator) && (
+          <Stack
+            horizontal
+            className={mergeStyles(
+              isVideoRendered ? videoHint : disabledVideoHint,
+              // when video is being rendered, the displayName has a grey-ish background, so no use of theme
+              { color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary },
+              styles?.displayNameContainer
             )}
+          >
+            <Stack>
+              {displayName && (
+                <Text
+                  className={mergeStyles(displayNameStyle, {
+                    color: isVideoRendered ? palette.neutralPrimary : theme.palette.neutralPrimary
+                  })}
+                >
+                  {displayName}
+                </Text>
+              )}
+            </Stack>
+            <Stack className={mergeStyles(iconContainerStyle)}>
+              {showMuteIndicator &&
```

Comment:
You probably want to drop the surrounding `Stack` (or `Stack.Item`, see my other comments) as well when you decided not to render the `Icon`, instead of rendering an empty `Stack`.